src/tinuous/__main__.py

```diff
@@ -128,8 +128,12 @@ def path_fields(self) -> Dict[str, str]:
             "common_status": COMMON_STATUS_MAP[self.status],
         }
 
-    def expand_path(self, path_template: str) -> str:
-        return path_template.format_map(self.path_fields())
+    def expand_path(self, path_template: str, vars: Dict[str, str]) -> str:
+        fields = self.path_fields()
+        expanded_vars: Dict[str, str] = {}
+        for name, template in vars.items():
+            expanded_vars[name] = template.format(**fields, **expanded_vars)
```

Comment:
right -- I "dreamed" about the bug in my suggested code ;-)  this should mitigate it. Thanks!